src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,94 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Basic todo task with only description
+ */
+public class Task {
+    protected String type;
+    protected String task;
+    private boolean isDone;
+    protected LocalDateTime endTime;
+
+    /**
+     * Constructor to a new todo task with default date
+     * of 1/1/0000 0000 so that it will be placed at the top when
+     * sorted
+     *
+     * @param task description of task
+     */
+    public Task(String task) {
+        this.task = task;
+        this.isDone = false;
+        this.type = "todo";
+        this.endTime = LocalDateTime.of(0, 1, 1, 0, 0);
```

Comment:
May want to consider converting the 0s and 1s to constants; so its easier for reviewers to understand your code